packages/graphql/src/utils/points.ts

```diff
@@ -0,0 +1,22 @@
+import { GetFeeTransfersQuery } from '../generated'
```

Comment:
I'm not sure we want to include this specific logic in the graphql package itself. 

We may want to keep this level of logic in the application layer and include only the pure queries/hooks in the package. We can talk more about it tomorrow.

Reply:
Heard, I can move it out of GraphQL package and into Launchpad. Good callout!